doc/conf.py

```diff
@@ -126,6 +126,11 @@
 ]
 
 
+# Configuration for sphinx.ext.autodoc
+# Do not expand following type aliases when generating the docs
+autodoc_type_aliases = {"color_like": "pyvista.color_like"}
```

Comment:
So despite the move this is still visible as `pyvista.color_like`, through a `_typing` -> `colors` -> `pyvista/plotting/__init__` -> `pyvista/__init__.py` chain. I think we can live with that (we could manually delete the name at the end of `colors.py` or something but probably not worth the effort).

Reply:
_But_ before your commits today the alias rendering didn't work:
![Screenshot from 2022-02-22 19-59-42](https://user-images.githubusercontent.com/17914410/155219947-7af6f721-52ad-4539-8f8e-b48dfe682e32.png)

I'll have to check your current PR state, but it's possible we have to use `pyvista._typing.color_like` here to get the alias to work correctly. Just a guess.

Reply:
Current version works beautifully!
![Screenshot from 2022-02-22 22-24-54](https://user-images.githubusercontent.com/17914410/155222074-45f4ace3-ba11-4ca3-9110-4d4ffdec44e0.png)


I guess `:data:` is OK too for the specification of `color_like` if we know it works (from the current PR state)

Reply:
All right. The reason it was no longer working before is that the `from __future__ import annotations` import is required in the module that is **using** the type alias, rather than only in the module where it is defined (`_typing`). That's a bit annoying, but as we are only using it in the `colors` module for now, not too much of a hassle.
Btw, I think I might have a solution for a direct `color_like` --> `Color` link in the docs, but maybe that's for a follow-up PR then.

Reply:
Good catch on the annotation import. I think it's perfectly fine to have there. And since the `color_like` docs mention `Color` explicitly there's a very findable link in that direction:
![Screenshot from 2022-02-23 11-30-14](https://user-images.githubusercontent.com/17914410/155302010-77d66b2b-cb7d-4924-ac03-af125a747132.png)

So I think this is great as-is.